tools/data-api-sdk/v1/models/sdk_constant_type.go

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package models
+
+// SDKConstantType defines the backing Type for a Constant used within the SDK.
+// In the majority of cases these are String values (StringSDKConstantType) however
+// both Float and Integer constants can exist too.
+type SDKConstantType string
```

Comment:
Would this perhaps be better as a type alias?
```suggestion
type SDKConstantType = string
```

Reply:
Agreed, good spot.